tests/integration/bucketlistController.int.test.js

```diff
@@ -103,7 +111,7 @@ describe("BucketList API Endpoints", () => {
     //Tests for authentication errors when trying to update a Bucketlist
     it("should return an error when a user is using an old token", async () => {
         const response = await request(app).put(baseEndpoint + id)
-            .set('Authorization', 'Bearer ' + "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVkZmJhNjI5NTJkNjliMTI2YzRmNGFjZCIsImlhdCI6MTU3Njc4MDY4NCwiZXhwIjoxNTc5MzcyNjg0fQ.39-8VdVXzTc6mHkxMCRHjGoY-KQo7cVYihD3tIjqek4")
+            .set('Authorization', 'Bearer ' + "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVlMDIyNTgzNzIwOWExMWNlMGE1YTlhYiIsImlhdCI6MTU3NzE5ODk4MiwiZXhwIjoxNTc5NzkwOTgyfQ.PF_FmBi_G1TCcpvklZ_UDqX7n5B-4RNl6kpNWq-A6Uc")
```

Comment:
Please move your test token to your env file. Every instance of it

Reply:
Okay, duly noted. 
I believe that code should be testable, both by unit and integration tests. 
I will work on my code again, and make sure it's testable by unit tests, not just integration tests.